### PR TITLE
chore: enable usdc message on buy with ach screen

### PR DESCRIPTION
**Pull request by dangelovska**

### CODE REVIEW

File: brd-ios/breadwallet/src/Redesign/Scenes/Buy/Main/BuyPresenter.swift [hunk 1]
```diff
@@ -202,6 +202,23 @@ final class BuyPresenter: NSObject, Presenter, BuyActionResponses {
         viewController?.displayMessage(responseDisplay: .init(error: error, model: model, config: config))
     }
     
+    func presentUSDCMessage(actionResponse: BuyModels.AchData.ActionResponse) {
+        let infoMessage = NSMutableAttributedString(string: L10n.Buy.disabledUSDCMessage)
+        let linkRange = infoMessage.mutableString.range(of: L10n.WalletConnectionSettings.link)
+        
+        if linkRange.location != NSNotFound {
+            infoMessage.addAttribute(.underlineStyle, value: 1, range: linkRange)
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Redesign/Scenes/Buy/Main/BuyPresenter.swift:210]: what is 1

File: brd-ios/breadwallet/src/Redesign/Scenes/Buy/Main/BuyViewController.swift [hunk 2]
```diff
@@ -304,7 +306,16 @@ class BuyViewController: BaseTableViewController<BuyCoordinator, BuyInteractor,
                                  configuration: responseDisplay.config)
     }
     
-    func displayAchData(actionResponse: BuyModels.AchData.ActionResponse) {
+    func displayManageAssets(actionResponse: BuyModels.AchData.ResponseDisplay) {
+        guard let superview = UIApplication.shared.windows.first(where: { $0.isKeyWindow }) else { return }
+        
+        let notification: FEInfoView = (superview.subviews.first(where: { $0 is FEInfoView }) as? FEInfoView) ?? FEInfoView()
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Redesign/Scenes/Buy/Main/BuyViewController.swift:312]: don't we already have something like that in Coordinator

File: brd-ios/breadwallet/src/Redesign/Scenes/Buy/Main/BuyInteractor.swift [hunk 3]
```diff
@@ -214,26 +214,33 @@ class BuyInteractor: NSObject, Interactor, BuyViewActions {
...
+            
+            switch dataStore?.paymentMethod {
+            case .buyAch:
+                if let currency = Store.state.currencies.first(where: { $0.code == C.USDC }) {
+                    dataStore?.toAmount = .zero(currency)
+                }
+                paymentCards = dataStore?.allPaymentCards?.filter { $0.type == .buyAch }
+                
+            default:
+                if let currency = Store.state.currencies.first(where: { $0.code == C.BTC }) ?? Store.state.currencies.first {
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Redesign/Scenes/Buy/Main/BuyInteractor.swift:230]: are you sure that Store.state.currencies.first is ok?

File: brd-ios/breadwallet/src/Redesign/Scenes/Buy/Main/BuyPresenter.swift (hunk 1 above)
Comment (dangelovska) [reply brd-ios/breadwallet/src/Redesign/Scenes/Buy/Main/BuyPresenter.swift:210]: this is true for underlineStyle 
Screenshot 2022-12-07 at 14 51 10

File: brd-ios/breadwallet/src/Redesign/Scenes/Buy/Main/BuyInteractor.swift (hunk 3 above)
Comment (dangelovska) [reply brd-ios/breadwallet/src/Redesign/Scenes/Buy/Main/BuyInteractor.swift:230]: yes, it's okay

**Review by kenanmamedoff — APPROVED** [review]

**Pull request merged**